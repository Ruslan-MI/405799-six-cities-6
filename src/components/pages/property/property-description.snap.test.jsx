import React from "react";
import {
  render,
} from "@testing-library/react";
import PropertyDescription from "./property-description";
import {
  mockOffer,
} from "../../../mocks-for-test";

it(`Property description should render correctly`, () => {

  const {
    container,
  } = render(<PropertyDescription description={mockOffer.description} />);

  expect(container).toMatchSnapshot();
});
