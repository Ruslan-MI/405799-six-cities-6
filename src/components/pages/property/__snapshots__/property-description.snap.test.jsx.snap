// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property description should render correctly 1`] = `
<div>
  <div
    class="property__description"
  >
    <p
      class="property__text"
    >
      A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.
    </p>
  </div>
</div>
`;
